doctr/utils/multithreading.py

```diff
@@ -25,11 +28,18 @@ def multithread_exec(func: Callable[[Any], Any], seq: Iterable[Any], threads: Op
 
     Returns:
         iterable of the function's results using the iterable as inputs
+
+    Notes:
+        This function uses ThreadPool from multiprocessing package, which
+        uses `/dev/shm` directory for shared memory.
+        If you do not have write permissions for this directory (like in a scenario when you run doctr on AWS Lambda),
```

Comment:
"like in a scenario when you run doctr on AWS Lambda" -> "if you run `doctr` on AWS Lambda for instance"